app/models/pglogical_subscription.rb

```diff
@@ -48,29 +50,34 @@ def self.save_all!(subscription_list)
     errors = []
     subscription_list.each do |s|
       begin
-        s.save!
+        s.save!(false)
       rescue => e
         errors << "Failed to save subscription to #{s.host}: #{e.message}"
       end
     end
 
+    EvmDatabase.restart_failover_monitor_service_queue
+
     unless errors.empty?
       raise errors.join("\n")
     end
     subscription_list
   end
 
-  def delete
+  def delete(reload_failover_monitor_service = true)
```

Comment:
Perhaps this should be `delete!` for consistency with `save!`

Reply:
Probably, but I don't think I want to change that in this PR.